ruby-challenges.rb

```diff
@@ -19,17 +25,62 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
-
+def sum (nums_array)
+  nums_array.inject(:+)
+end
+p sum(nums_array1)
+p sum(nums_array2)
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
-# Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
-
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+      def initialize(model, wheel, current_speed)
+          @model = model
+          @wheel = 2
+          @current_speed = current_speed
+      end
+      def my_bike (model, wheel, current_speed)
+         "The #{model} bike has #{wheel} wheels and is going #{current_speed} mph"
+      end
+end
 
+  bike_info = Bike.new('Trek', 2, 0)
+# p bike_info.my_bike('Trek', 2, 0)
 
 # -------------------3b) Add the ability to pedal faster and brake. The pedal_faster method should increase the speed by a given amount. The brake method should decrease the speed by a given amount. The bike cannot go negative speeds.
 
 # Expected output example: my_bike.pedal_faster(10) => 10
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+      def initialize(model, wheel, current_speed)
+          @model = model
+          @wheel = 2
+          @current_speed = current_speed
+      end
+      def my_bike (model, wheel, current_speed)
```

Comment:
No need for my_bike method to have params as you have access to the instance variables within your class.